feature_layers/dynamic-entity-layer/src/main/java/com/esri/samples/dynamic_entity_layer/DynamicEntityLayerSample.java

```diff
@@ -0,0 +1,307 @@
+
+      // create a map view and set the map to it
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // create an envelope around Sandy, Utah to be the extent used by the dynamic entity filter and viewpoint
+      var utahSandyEnvelope = new Envelope(new Point(-112.110052, 40.718083, SpatialReferences.getWgs84()),
+        new Point(-111.814782, 40.535247, SpatialReferences.getWgs84()));
+
+      // set the viewpoint to the map view
+      mapView.setViewpoint(new Viewpoint(utahSandyEnvelope));
+
+      // create a graphics overlay and add a graphic to show the envelope's extent using a simple line symbol
+      var borderLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DASH, Color.RED, 2);
+      var graphicsOverlay = new GraphicsOverlay();
+      var graphic = new Graphic(utahSandyEnvelope, borderLineSymbol);
+      graphicsOverlay.getGraphics().add(graphic);
+
+      // add the graphics overlay to the map view
+      mapView.getGraphicsOverlays().add(graphicsOverlay);
+
+      // create a controls vbox containing the UI components
+      setUpControlsVBox();
+
+      // create the stream service as a dynamic entity data source
+      var streamServiceUrl = "https://realtimegis2016.esri.com:6443/arcgis/rest/services/SandyVehicles/StreamServer";
+      var streamService = new ArcGISStreamService(streamServiceUrl);
+
+      // add a filter for the data source to limit the amount of data received by the application
+      var streamServiceFilter = new ArcGISStreamServiceFilter();
+      streamServiceFilter.setGeometry(utahSandyEnvelope);
+      streamServiceFilter.setWhereClause("speed > 0");
+      streamService.setFilter(streamServiceFilter);
+
+      // set a duration of five minutes for how long observation data is stored in the data source
+      streamService.getPurgeOptions().setMaximumDuration(5);
+
+      // create a layer to display the data from the stream service
+      dynamicEntityLayer = new DynamicEntityLayer(streamService);
+
+      // create unique value renderers for customizing the dynamic entity layer appearance
+      var uniqueValueRenderers = createUniqueRenderers();
+
+      // set the dynamic entity and previous observation renderers
+      dynamicEntityLayer.setRenderer(uniqueValueRenderers.get(0));
+      dynamicEntityLayer.getTrackDisplayProperties().setPreviousObservationRenderer(uniqueValueRenderers.get(1));
+
+      // create a simple line renderer and set the track line renderer to it
+      var trackLineRenderer = new SimpleRenderer(new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.LIGHTGRAY, 2));
+      dynamicEntityLayer.getTrackDisplayProperties().setTrackLineRenderer(trackLineRenderer);
+
+      // update the label with the service connection status
+      connectionStatusLabel.textProperty().bind(Bindings.createStringBinding(() ->
+        "Status: " + streamService.connectionStatusProperty().asString().get(), streamService.connectionStatusProperty()));
```

Comment:
```suggestion
        "Status: " + streamService.connectionStatusProperty().getValue(), streamService.connectionStatusProperty()));
```
Looks like you can use `getValue()` to clean this up